code/presentation/mb-client/package.json

```diff
@@ -46,6 +46,7 @@
     "@react-navigation/bottom-tabs": "^6.5.7",
     "@react-navigation/native": "^6.1.6",
     "@react-navigation/native-stack": "^6.9.12",
+    "@react-navigation/stack": "^6.3.25",
```

Comment:
do we need this? we are already using native-stack

Reply:
Removed